### PR TITLE
Introduce e2e testing

### PR DESCRIPTION
**Pull request by AlexeyKasatkin**
Introduce e2e testing.
Add a simple e2e test.
Fix dependencies.

Related: https://github.com/Mirantis/k8s-netchecker-server/issues/81

This change is [Reviewable](https://reviewable.io/reviews/mirantis/k8s-netchecker-server/99)

### CODE REVIEW

**Comment by xenolog** [comment]
+1

**Comment by xenolog** [comment]
Reviewed 6 of 6 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-netchecker-server/99)*

File: test/e2e/basic_test.go [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+		},
+	}
+
+}
+
+func newDeployment(deploymentName string, replicas int32, podLabels map[string]string, imageName string, image string, cmd []string, env []v1.EnvVar) *v1beta1.Deployment {
+	return &v1beta1.Deployment{
+		ObjectMeta: meta_v1.ObjectMeta{Name: deploymentName},
+		Spec: v1beta1.DeploymentSpec{
+			Replicas: &replicas,
```
Comment (falkerson) [inline test/e2e/basic_test.go]: why we set reference here?

Comment (AlexeyKasatkin) [inline test/e2e/basic_test.go]: It is just to be in accordance with DeploymentSpec specification: "Replicas *int32".

**Pull request merged**